### DISCUSSION TITLE
zonebudget for MF6 works on on gwt?

### DISCUSSION THREAD

**Discussion by RyanConway91** [MODFLOW-ORG/modflow6#1181] [General]
I tried running zonebudget on a cbt file from a gwt. It seems to work, however the documentation for zb did not make it clear that it can be ran on gwt outputs. 

Is this safe to do? If so, I suggest updating the documentation to indicate this. If not, likewise!

**Comment by christianlangevin** [comment]
Hey @RyanConway91, thanks for the message here.  Yes, zonebudget is intended to work, by design, on budget files from GWF and GWT.  We will make a note of this in the documentation.

**Comment by wpbonelli** [comment]
Noted in zonebudget docs with https://github.com/MODFLOW-USGS/modflow6/pull/1195.
